src/main/resources/schema.graphqls

```diff
@@ -0,0 +1,25 @@
+type Run {
+    runId: String!
+    analysisId: String
+    state: String
+    workDir: String
+    #    request: RunRequest
+}
+
+scalar JSON
+type RunRequest {
+    workflowUrl: String!
+    workflowParams: JSON!
+    workflowEngineParams: WorkflowEngineParams
+}
+
+type WorkflowEngineParams {
+    revision: String
+    resume: String
+    workDir: String
+    latest: Boolean
+}
```

Comment:
if you want to make this feature complete against what is being simulated in `workflow-ui` check this out: https://github.com/icgc-argo/workflow-ui/blob/develop/src/gql/typedefs.ts